datajoint/fetch.py

```diff
@@ -199,15 +153,11 @@ def __getitem__(self, item):
         :return: tuple with an entry for each element of item
 
         Examples:
-
         >>> a, b = relation['a', 'b']
         >>> a, b, key = relation['a', 'b', datajoint.key]
-        >>> results = relation['a':'z']    # return attributes a-z as a tuple
```

Comment:
Why is that being deleted? Did you delete the slicing functionality? If so, why?